.pre-commit-config.yaml

```diff
@@ -1,7 +1,8 @@
 repos:
--   repo: https://github.com/dotnet/format
-    rev: v5.1.225507
+-   repo: local
     hooks:
     -   id: dotnet-format
-        entry: dotnet-format Source/Dafny.sln -w --
-        
\ No newline at end of file
+        name: dotnet-format
+        language: system
+        entry: dotnet tool run dotnet-format --folder -w --include
```

Comment:
I guess it worked before, but this is more efficient as it doesn't format all the files in the solution?

Reply:
It's more that `pre-commit` always provides the list of changed files whether you want them or not. :) It seems like using `dotnet tool run` makes the command not work unless it uses `--include`, because otherwise it tries to parse the files as options.